WakaTime/Watcher.swift

```diff
@@ -46,7 +44,7 @@ class Watcher: NSObject {
     private func handleAppChanged(_ app: NSRunningApplication) {
         if app != activeApp {
             NSLog("App changed from \(activeApp?.localizedName ?? "nil") to \(app.localizedName ?? "nil")")
-            if let newAppID = app.bundleIdentifier, Watcher.appIDsToWatch.contains(newAppID) {
+            if let newAppID = app.bundleIdentifier, MonitoringManager.isAppMonitored(for: newAppID) {
```

Comment:
We should change `setXcodeVersion(app)` below to update a hash of versions for bundle ids so we don't accidentally set Xcode's version to Postman's version. Then when sending the heartbeat and building the `User-Agent` string, we use the corresponding version for the heartbeat's bundle id. We should also change the name of the app in that string to match the app we're tracking.